### ISSUE TITLE
Cant build APK file

### ISSUE THREAD

**Issue by Juliuspy** [schul-cloud/schulcloud-flutter#348]
When I try to build the apk, It gives me this error:
* What went wrong:
Execution failed for task ':app:validateSigningScRelease'.
> Keystore file 'C:\Users\Username\Contacts\schulcloud-flutter\android\upload.keystore.jks' not found for signing config 'release'.

**Comment by JonasWanke** [comment]
When building a release build, a secret key is required for signing the generated `.apk`. You have two options:

- Set the environment variable `RELEASE_USE_DEBUG_KEY` to `true` to reuse the debug signing key that Android Studio generates automatically. (This is the simpler option that should be sufficient.)
- [Generate a custom key](https://developer.android.com/studio/publish/app-signing#generate-key) and use that by placing the key file in the location mentioned above and updating the `storePassword`, `keyAlias` & `keyPassword` in `android/app/build.gradle` in the `signingConfigs` section.

Note that you can also download an auto-generated `.apk` signed with a debug key from our CI by visiting the [Actions tab](https://github.com/hpi-schul-cloud/schulcloud-flutter/actions?query=branch%3Amaster+), visiting the latest (top-most) run, selecting any of the "Build"-links on the left, opening the "Artifacts" dropdown on the top-right, and selecting your desired `apk` flavor.

Please let me know if this worked and feel free to ask if you have any further questions :)

**Comment by Juliuspy** [comment]
Thank You very much.
It worked and I have installed the app succesfully.
But I have another issue:
When I open the app and go to "Kurse" and then to my course it says "Diese Ressource konnte nicht geladen werden".
Do you have any solutions for that?

Again Thank You very much. :)

**Comment by JonasWanke** [comment]
Unfortunately, that's a known issue that will take some time to resolve. Due to security reasons, the API routes of the server were blocked a while ago and are only unblocked little by little. We're still waiting for the server teams to unblock the remaining ones.  
This error currently occurs when trying to load assignments, calendar entries, users, and, on some flavors, courses.

**Comment by Juliuspy** [comment]
Thank You very much
